source/collaborate/make-calls.rst

```diff
@@ -116,6 +122,20 @@ To record a call:
   
   To start recording, use the ``/call recording start`` slash command. When you're finished recording, use the ``/call recording stop`` slash command. Alternatively, expand the call window using the arrows in the top-right of the active call banner. Then select the **Record** button. To finish, select the **Record** button again.
 
+Live captions during calls
+---------------------------
+
+.. include:: ../_static/badges/ent-only.rst
+  :start-after: :nosearch:
+
+From Mattermost v9.7, when recording calls, and when :ref:`live captions are enabled <configure/plugins-configuration-settings:enable live captions (experimental)>`, all call participants can display real-time text captions by selecting the **Gear** |gear-icon| icon and **Show live captions**. Live captions can be helpful in cases where noise is preventing you from hearing other participants clearly.
```

Comment:
@streamer45 - Are live captions supported via mobile? If so, what version was this supported added to?

Reply:
Yes, this was added to 2.16. You need to click the three dot `More` button, and select `Show live captions`.